### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by vkresch**
Adding three types of issue templates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
```
Comment (jdsika) [inline .github/ISSUE_TEMPLATE/bug_report.md]: 'bug'   ???

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
```
Comment (jdsika) [inline .github/ISSUE_TEMPLATE/feature_request.md]: I reworked the labels - can you copy/paste them for all the other repositories and review them?

this one should be 'feature request'

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (jdsika) [inline .github/ISSUE_TEMPLATE/feature_request.md]: dot

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
```
Comment (jdsika) [inline .github/ISSUE_TEMPLATE/feature_request.md]: you have - please do not use abbreviations in written english

File: .github/ISSUE_TEMPLATE/question.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
+---
+name: Question
+about: Ask a question if you don't understand something
+title: ''
+labels: question
```
Comment (jdsika) [inline .github/ISSUE_TEMPLATE/question.md]: 'question' ?

File: .github/ISSUE_TEMPLATE/question.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
+---
+name: Question
+about: Ask a question if you don't understand something
```
Comment (jdsika) [inline .github/ISSUE_TEMPLATE/question.md]: do not

**Comment by jdsika** [comment]
@vkresch Do you get an e-mail notification with my review or do I need to ping you in the conversation?

**Comment by vkresch** [comment]
@jdsika Yes I got your review I'm just experimenting also with pull requests templates.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (vkresch) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I tested and it is bug and not 'bug'

**Comment by jdsika** [comment]
Just a last thing: Decide if you want dots, double dots, or questionmarks or blank at the end of a headline. The first headlines are blank, all others not?

**Pull request merged**